app/javascript/mastodon/locales/en.json

```diff
@@ -56,6 +57,7 @@
   "column_header.unpin": "Unpin",
   "column_subheading.navigation": "Navigation",
   "column_subheading.settings": "Settings",
+  "compose_form.direct_message_warning": "This toot will only be visible to all the mentioned users.",
```

Comment:
I like this.

Due to some recent discussions, I think we might need to include a warning not to exchange dangerous information in DMs because DMs are stored on the server and can be read by admins. I'm not sure there's a consensus on this yet, but this would be the place to put it if we decide to.

Reply:
We've had a few similar discussions on switter.at, although haven't come to a conclusion yet. I think it should just be assumed that admins potentially can read direct messages for moderation purposes, they're not private messages, they're direct messages.

The inspiration behind this warning was accidentally direct messaging an abuser when dm'ing with a abuse reporter, which caused a lot of small issues.

Reply:
Also, the wording is a bit awkward here, but I can't think of a better way to word it. I want to make sure it's stating the visibility and that ALL the users mentioned will see it.

Reply:
Right. Okay. I was gonna post a strawpoll about that to gather feedback, but mastodon.social went down due to Docker issues.

Re: awkward wording, yes, the "only" followed by "all the" is a bit odd. Is "all the" necessary? "This toot will only be visible to mentioned users" -> If I mention someone, they will see it.

Reply:
It's more, I expected it to only be the first mention, i.e., in:

```
@switter I've been having issues with @lolahunt, they're so aussie
```

That going to lolahunt & switter, when I only meant to send to switter is an issue. 

Where as if I did this, I'd expect it to go to both:
```
@switter @lolahunt really loving switter! thank you!
```

(lola, you're absolutely fine promise)

Reply:
So I wanted a way to communicate that it's ANY mention in the Toot.

Reply:
How about "The toot will be sent to all mentioned users"?

Reply:
hm, yeah, but that doesn't indicate it's "private"